network/interface.go

```diff
@@ -124,4 +124,7 @@ type Network interface {
 	NumConnectedPeers() int
 	ReachabilityStatus() string
 	HostAddrs() []string
+	Name() string
```

Comment:
Name is always fix and won't change. I am not sure why you make an API for that.

Reply:
I considered if someone run a private/local network with pactus protocol, so probably they will have a different name. also a fork of pactus can have a different name for the network.

Reply:
Hoom, not really a good reason to add another API.

Reply:
we can remove it. but I belive having network name can help. also specifing the network with chain ID or something like that can help in future. this kind of stuff helps us in different golas in future.

Reply:
This is fixed per chain. We can even get it from the config. Check config [here](https://github.com/pactus-project/pactus/blob/440761955621b04f7a6eb484fe9c3dd1d03065d8/config/config.go#L121C2-L121C2)

Reply:
yes, I knew we can change it in config per chain also. this function is just a API to show this in network info method, because it's  **Network** info. so I think we should also get the network name from it. for example a same SDK can call different pactus networks (testnet/mainnet) and also different pactus forks, and also detect which network it's calling with this parameter.

Reply:
Ok, I am convinced. Let's add `Name` API to the network module.